tools/publish-verification-info.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+verification_dir="$1"
+img="$1/verification.raw"
+ip="$2"
+
+if [ "$verification_dir" == "" ] || [ "$ip" == "" ]; then
+  echo "Usage ./publish_verification_info.sh <verification_dir> <server_ip>"
+  exit
+fi
+
+size=$(fdisk -l "$img" | grep raw5 | awk '{print $2}')
+mkdir /tmp/verification_mnt
+sudo mount -o offset=$((size*512)) "$img" /tmp/verification_mnt
+
+dirname=$(cat /tmp/verification_mnt/*/hardwaremodel.txt | grep "Product Name:" | awk '{ for (i = 3; i <= NF; i++) printf "%s", $i; printf "\n" }' | tr ',' '-' | tr '(' '-' | tr ')' '-' | tr '+' '-')
+eve_version=$(cat "${verification_dir}/verification/eve_version")
+dirname="${dirname}###${eve_version}"
+
+mkdir "$dirname"
+cp -r /tmp/verification_mnt/*/* "$dirname/"
+tar xvf "${dirname}/hw.info.txz" -C "$dirname"
+rm "$dirname/hw.info.txz"
+
+fname="${dirname}.tar.gz"
+tar zcvf "${fname}" "$dirname"
+rm -rf "$dirname"
+
+CSRF_TOKEN=$(curl -s -c cookies.txt "http://$ip:8999/upload" | awk -F"\"" '{if (NR==2) print $6}')
```

Comment:
Looks tricky.
Is the server under our control, and thus, we are sure the response can always be parsed this way?
Or we could use some parsing tool that contains less hardcoded values. Something like
```bash
CSRF_TOKEN=$(curl -s -c cookies.txt "http://$ip:8999/upload" | xmllint --html --xpath 'string(//input[@name="csrfmiddlewaretoken"]/@value)' - 2>/dev/null)

```

Reply:
Sure, I will do what you propose.

Reply:
So, it works? I just tried to guess, how the HTTP response could look like and didn't expect it to be exactly like this))

Reply:
Good)))